src/ObjCRuntime/Registrar.core.cs

```diff
@@ -1,10 +1,16 @@
 using System.Text;
 
+#nullable enable
+
 namespace Registrar {
 	abstract partial class Registrar {
-		internal static string CreateSetterSelector (string getterSelector)
+		internal static string? CreateSetterSelector (string? getterSelector)
 		{
+#if NET
 			if (string.IsNullOrEmpty (getterSelector))
+#else
+			if (getterSelector is null || string.IsNullOrEmpty (getterSelector))
```

Comment:
Why do we need the 'getterSelector is null' here? Does the string.IsNullOrEmpty not catch that case in legacy?